.gitignore

```diff
@@ -36,3 +36,6 @@ avalanche-*.tar
 
 # Flamegraph Files
 *.bggg
+
+# Elixir LS
+/.elixir_ls/
```

Comment:
these types of entries and editor specific entries in `.gitignore` belong in a global `.gitignore`. Here is some info about it: https://github.com/elixir-lang/elixir/pull/7527#discussion_r178758439 and here is how to set one up: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#create-a-global-gitignore

Reply:
GitHub default `.gitignore` doesn't agree. :)
https://github.com/github/gitignore/blob/main/Elixir.gitignore

Why does it matter that it is global and not in the repo?

Reply:
I can get on board with this:

> it is not the project maintainer’s responsibility to ignore files that come from your work environment — it’s yours. These files do not belong in the project’s .gitignore file because they’re specific to you, not to the project.

Reply:
Github is great, but I'd defer that judgment to how the Elixir ecosystem handles these. I'll be honest, I didn't even know that was a thing 😂 . If you find one of those in the Elixir ecosystem managed by Jose, lunch is on me.

This is just a nit, but definitely something folks look at in open source projects. It's a slippery slope and folks have learned to not deal with it. Otherwise you get `.swp`s, `DS_Store`s, `whatever_your_editor_is` and others in `gitignores`. Configuring a global is simple enough. This is also true for in house projects, but that's a different conversation 😂 

No action needed here.

I'm actually going to edit/delete the comment above so this doesn't get linked to that Elixir PR. Joy of building in the open haha